### ISSUE TITLE
No pokestops

### ISSUE THREAD

**Issue by kassumi** [NoxxDev/NecroBot#739]
So i thought i get ip ban, but i wanted to check to be sure, on necro, it says there is no pokestops in new york, when i log in from my phone, i saw mamy pokestops in Wrocław, i even catched a Pokemon (i used same Wi-Fi my lpc is using). So my question iż, whats wrong with necro?

**Comment by tanjj92** [comment]
maybe DeviceID banned that you used in necro?

**Comment by kassumi** [comment]
nope, i fixed it, it turnd ot that the location was actually pokestopless xD. Anyway, to not start another issu, where can i post ideas for necrobot?
